src/components/CreatePollModal/CreatePollModal.jsx

```diff
@@ -0,0 +1,125 @@
+      >
+        <div
+          className={`d-flex playlist-li-item videolib-list-container border-bottom j-space-between mb-small`}
+        >
+          <h3 className="break-word">Create a poll</h3>
+          <i
+            className="fa-solid fa-rectangle-xmark cursor-p"
+            onClick={() => setPollModal((prev) => !prev)}
+          ></i>
+        </div>
+
+        <div className="poll-modalbody-container">
+          <div className="d-flex flex-direction-col playlist-li-item cursor-p gap-small mb-small">
+            <label className="profile-edit-modal-inp-label">
+              Your question
+            </label>
+
+            <TextareaAutosize
+              type="text"
+              placeholder="E.g., How do you commute to work?"
+              className="profile-edit-modal-inp"
+              onChange={(e) =>
+                setPollData({ ...pollData, question: e.target.value })
+              }
+            />
+          </div>
+
+          <div className="d-flex flex-direction-col playlist-li-item cursor-p gap-small mb-small">
+            <label className="profile-edit-modal-inp-label">Option 1</label>
+            <input
+              type="text"
+              placeholder="E.g., Public transportation"
+              className="profile-edit-modal-inp"
+              onChange={(e) =>
+                setPollData({ ...pollData, option1: e.target.value })
+              }
+            ></input>
+          </div>
+
+          <div className="d-flex flex-direction-col playlist-li-item cursor-p gap-small mb-small">
+            <label className="profile-edit-modal-inp-label">Option 2</label>
+            <input
+              type="text"
+              placeholder="E.g., Drive myself"
+              className="profile-edit-modal-inp"
+              onChange={(e) =>
+                setPollData({ ...pollData, option2: e.target.value })
+              }
+            ></input>
+          </div>
+
+          {pollModalOptionNum >= 3 && (
+            <div className="d-flex flex-direction-col playlist-li-item cursor-p gap-small mb-small">
+              <label className="profile-edit-modal-inp-label">Option 3</label>
+              <input
+                type="text"
+                placeholder="E.g., Ola or Uber"
+                className="profile-edit-modal-inp"
+                onChange={(e) =>
+                  setPollData({ ...pollData, option3: e.target.value })
+                }
+              ></input>
+            </div>
+          )}
+
+          {pollModalOptionNum == 4 && (
+            <div className="d-flex flex-direction-col playlist-li-item cursor-p gap-small mb-small">
+              <label className="profile-edit-modal-inp-label">Option 4</label>
+              <input
+                type="text"
+                placeholder="E.g., Other"
+                className="profile-edit-modal-inp"
+                onChange={(e) =>
+                  setPollData({ ...pollData, option4: e.target.value })
```

Comment:
you can extract this out in function and setPollData from a single function
Something like this
`const pollDataHandler = (e) => {
setPollData(prev => ({...prev,[e.target.name]:e.target.value}))
}`

Reply:
Great idea , I will do that